src/coprocessor/codec/convert.rs

```diff
@@ -153,13 +321,6 @@ pub fn bytes_to_uint_without_context(bytes: &[u8]) -> Result<u64> {
     r.ok_or_else(|| Error::overflow("BIGINT UNSIGNED", ""))
 }
 
-/// `bytes_to_int` converts a byte arrays to an i64 in best effort.
-pub fn bytes_to_int(ctx: &mut EvalContext, bytes: &[u8]) -> Result<i64> {
-    let s = str::from_utf8(bytes)?.trim();
-    let vs = get_valid_int_prefix(ctx, s)?;
-    bytes_to_int_without_context(vs.as_bytes()).map_err(|_| Error::overflow("BIGINT", &vs))
-}
-
 /// `bytes_to_uint` converts a byte arrays to an u64 in best effort.
 pub fn bytes_to_uint(ctx: &mut EvalContext, bytes: &[u8]) -> Result<u64> {
```

Comment:
The implementation of this function looks to be incorrect.

Reply:
The function will be addressed in the next PR (convert * to uint).